ingestion/functions/parsing/new_zealand/new_zealand.py

```diff
@@ -0,0 +1,135 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            'common/python'))
+    import parsing_lib
+
+import json
+import os
+import sys
+from datetime import date, datetime
+import csv
+
+
+def convert_date(raw_date: str, dataserver=True):
+    """
+    Convert raw date field into a value interpretable by the dataserver.
+
+    Removing timestamp as always midnight.
+
+    Set dataserver to False in order to return version appropriate for notes.
+    """
+    date = datetime.strptime(raw_date.split(' ')[0], "%Y-%m-%d")
+    if not dataserver:
+        return date.strftime("%m/%d/%Y")
+    return date.strftime("%m/%d/%YZ")
+
+
+def convert_gender(raw_gender):
+    if raw_gender == "Male":
+        return "Male"
+    if raw_gender == "Female":
+        return "Female"
+    return None
+
+
+def convert_location(raw_entry):
+    if 'Managed Isolation' in raw_entry['DHB']:
+        return None
```

Comment:
Should this return `{"query": "New Zealand"}` rather than `None`? I believe that a location is required or the insertion will fail.

Reply:
@calremmel They're cases coming from 'Overseas', but they don't specify where. Following your suggestion, I've changed it so that Overseas cases arriving in New Zealand are geocoded as New Zealand, but this might be something we want to discuss. @AnyaLindstromBattle for visibility.

Reply:
Sounds great! Thanks Felix. This looks good to me now as soon as you give me the go ahead on this point.

Reply:
Yep, that's fine from my side